sources/packages/backend/libs/test-utils/src/factories/supporting-user.ts

```diff
@@ -0,0 +1,46 @@
+import { Application, SupportingUser, SupportingUserType } from "@sims/sims-db";
+import { DataSource } from "typeorm";
+
+/**
+ * Creates a fake supporting user.
+ * @param supportingUserType supporting user type to be created.
+ * @param relations dependencies:
+ * - `application`: application that the supporting user is associated with.
+ * @returns a fake supporting user.
+ */
+export function createFakeSupportingUser(
+  supportingUserType: SupportingUserType,
```

Comment:
Providing `supportingUserType` here would be the same case that we discussed in the past.
If there is a Team desire we can revisit the topic.